### DISCUSSION TITLE
Auto update

### DISCUSSION THREAD

**Discussion by dogancanbakir** [projectdiscovery/pdtm#32] [Ideas]
### Please describe your feature request:
Auto-update feature as an option to enable/disable automatic updates of the tools installed on the system.

### Describe the use case of this feature:
With the Auto-update feature, pdtm can automatically update the PD tools with the latest updates and enhancements. Users no longer have to keep looking for recent updates constantly. This might be a problem for specific scenarios, like when there is a breaking change. The auto-update feature can only be allowed for a subset of tools that won't cause a problem. That's something we need to decide. 

Auto-update can have a time option for the update time, like non-working hours, etc.

**Comment by MetzinAround** [comment]
@dogancanbakir is this something you're still considering adding, or has already been added?

**Reply by dogancanbakir** [reply]
@MetzinAround, I still think we should add this as a feature, but not sure whether this is already added/tracked/etc.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
